File: scaffold/src/main/java/top/xuqingquan/web/agent/QuickCallJs.java
package top.xuqingquan.web.agent;

public interface QuickCallJs {

    void quickCallJs(String method, android.webkit.ValueCallback<String> callback, String... params);

    void quickCallJs(String method, com.tencent.smtt.sdk.ValueCallback<String> callback, String... params);

    void quickCallJs(String method, String... params);

    void quickCallJs(String method);
}

File: scaffold/src/main/java/top/xuqingquan/web/x5/QuickCallJs.kt
package top.xuqingquan.web.x5

import com.tencent.smtt.sdk.ValueCallback
import top.xuqingquan.web.agent.QuickCallJs

interface QuickCallJs :QuickCallJs{

    fun quickCallJs(method: String, callback: ValueCallback<String>, vararg params: String)

}
